modules/account-lib/README.md

```diff
@@ -1,61 +1,95 @@
-|Stellar|xlm|txlm|Not yet...|
+
+Below is the list of coins supported by this library -- as well as those that
+are on the roadmap.
+
+| Coin             | Mainnet Ticker | Testnet Ticker | Supported In Library |
+| :--------------- | :------------- | :------------- | :------------------- |
+| Tron             | trx            | ttrx           | Yes                  |
+| Tezos            | xtz            | txtz           | Yes                  |
+| Ethereum         | eth            | teth           | Yes                  |
+| CELO             | celo           | tcelo          | Yes                  |
+| Ethereum Classic | etc            | tetc           | Yes                  |
+| RSK              | rbtc           | trbtc          | Yes                  |
+| EOS              | eos            | teos           | Not yet...           |
+| Alogrand         | algo           | talgo          | Not yet...           |
+| Ripple           | xrp            | txrp           | Not yet...           |
+| Stellar          | xlm            | txlm           | Not yet...           |
 
 ## Core Concepts
 
 ### TransactionBuilder
-The `TranssactionBuilder` class guides a user through the construction of a transaction. The purpose of the `TransactionBuilder` is to yield `Transaction` object that is able to be broadcast to the network.
+
+The `TranssactionBuilder` class guides a user through the construction of a
+transaction. The purpose of the `TransactionBuilder` is to yield a `Transaction`
+object that can broadcast to the network.
 
 ### Transaction
-`Transaction` objects are javascript representations of blockchain transactions that implement protocol specific validation rules. `Transactions` provide encoding mechanisms that allow them to be validly broadcast to their respective network.
+
+`Transaction` objects are JavaScript representations of blockchain transactions
+that implement protocol specific validation rules. `Transactions` provide
+encoding mechanisms that allow them to be validly broadcast to their respective
+network.
 
 #### Transaction Types
-The `TransactionBuilder` supports 3 types types of transactions
+
+The `TransactionBuilder` supports the following transaction types:
+
+- [Send](#send)
+- [Wallet Initialization](#wallet-initialization)
+- [Address Initialization](#address-initialization)
+- [Account Update](#account-update)
+
 ##### Send
-Transfers funds from a wallet
+
+Transfers funds from a wallet.
+
 ##### Wallet Initialization
-Initializes a wallet's account on the network (e.g. Multi-sig contract deployment)
+
+Initializes a wallet's account on the network (e.g., multi-sig contract
+deployment).
+
 ##### Address Initialization
-Initializes a wallet's address on the network (e.g. Forwarder contract deployment)
+
+Initializes a wallet's address on the network (e.g., forwarder contract
+deployment).
+
 ##### Account Update
-Updates an account on the network (e.g. Public key revelation operation for Tezos)
 
-### Offline Availability
```

Comment:
this still seems relevant to highlight, that the SDK can function in an offline environment